simulator/core/perf_sim.cpp

```diff
@@ -315,6 +340,13 @@ void PerfSim<ISA>::clock_writeback( Cycle cycle)
     /* perform writeback */
     rf->write_dst( instr);
 
+    /* check for bubble */
+    if(instr.is_bubble())
+    {
+        ++executed_instrs;
```

Comment:
Looks like only bubbles got executed since there is no writes to `wp_target`, so you got an infinite loop.